app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -803,6 +826,39 @@ Vue.mixin({
           vm.fulfillment_context.active_step = 'payout_amount';
           break;
       }
+    },
+    showWorkSubmitted: function(handle) {
```

Comment:
this is not being used anywhere right ?

Reply:
oops... now i'm tryna figure out why this is here... looks like important stuff though

Reply:
that was reverted ! i'm assuming a merge / rebase at your end when wrong 😓 
https://github.com/gitcoinco/web/pull/8790
^ could you check that PR and remove that piece of code

Reply:
done https://github.com/gitcoinco/web/pull/8550/commits/9af812618e10373350424e16e3663fb61cc2ee05